test/integration/coverage_test.dart

```diff
@@ -38,6 +41,10 @@ Future<bool> runCoverage(String projectPath, {bool html: false}) async {
   }
 
   List args = ['run', 'dart_dev', 'coverage'];
+  if (functional) {
+    args.add('--functional');
+    args.add('--no-unit');
```

Comment:
Why should including the functional test suite prevent us from running unit tests?